### PR TITLE
fix bug 1119532 - add honeypot field to account signup form

### PR DESCRIPTION
**Pull request by groovecoder**

### CODE REVIEW

File: kuma/users/views.py [hunk 1]
```diff
@@ -418,5 +419,11 @@ def get_context_data(self, **kwargs):
         })
         return context
 
+    def dispatch(self, request, *args, **kwargs):
+        ret = verify_honeypot_value(request, None)
+        if type(ret) is HttpResponseBadRequest:
```
Comment (jezdez) [inline kuma/users/views.py]: `isinstance` may be better

**Comment by groovecoder** [comment]
Updated to use `isinstance` instead of `type`

File: kuma/users/views.py [hunk 2]
```diff
@@ -418,5 +419,11 @@ def get_context_data(self, **kwargs):
         })
         return context
 
+    def dispatch(self, request, *args, **kwargs):
+        ret = verify_honeypot_value(request, None)
```
Comment (jezdez) [inline kuma/users/views.py]: `s/ret/response/g`

**Comment by jezdez** [comment]
LGTM otherwise

File: kuma/users/views.py (hunk 2 above)
Comment (groovecoder) [inline kuma/users/views.py]: I had `response` in there at first, but then I saw that `verify_honeypot_value` can also return `None`, so I changed it. I.e., the returned value may not be a response, so I just named it `ret`. I could go either way on it though. I'll leave it up to you.

Comment (jezdez) [inline kuma/users/views.py]: a `response` can be None, that's alright

**Comment by groovecoder** [comment]
Updated to `response` instead of `ret`. Just need Travis to pass ...

**Comment by jezdez** [comment]
Yay!

**Pull request merged**